src/UKHO.ERPFacade.Common/Infrastructure/IEnumerableExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+
+namespace UKHO.ERPFacade.Common.Infrastructure
+{
+    [ExcludeFromCodeCoverage]
+    public static class IEnumerableExtensions
```

Comment:
Is this correct?

Assume the name is misleading?

Reply:
yes. It does sound misleading. I renamed the IEnumerableExtensions class to IEnumerableComparerExtension.